docs/source/changelog.rst

```diff
@@ -21,10 +21,36 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
     ``pip --version``.
 
 
+.. _release-5.1.0:
+
+5.1.0
+-----
+
+- Preliminary i18n implementation (:ghpull:`2140`).
+- Expose URL with auth token in notebook UI (:ghpull:`2666`).
+- Fix search background style (:ghpull:`2387`).
+- List running notebooks with ``--allow-root`` (:ghpull:`2421`).
```

Comment:
I think the point of this PR is that you can list running notebook servers without needing `--allow-root`, whereas previously it was doing that check when it wasn't necessary.